e2e-tests/controller-spark/controller_spark_sql_validation.sh

```diff
@@ -200,11 +197,8 @@ function wait_for_completion() {
 
   while [[ $(date -u +%s) -le ${end_time} ]]
   do
-    local pipeline_status=$(curl --location --request GET "${PIPELINE_CONTROLLER_URL}/status?" \
-    --connect-timeout 5 \
-    --header 'Content-Type: application/json' \
-    --header 'Accept: */*' -v \
-    | jq -r '.pipelineStatus')
+    local pipeline_status=$(controller "${PIPELINE_CONTROLLER_URL}" status \
+    | sed -n '/{/,/}/p' | jq -r '.pipelineStatus')
```

Comment:
Out of curiosity, why do we now need the pattern match command (`sed`) before sending the output to `jq`?

Reply:
That's because the output is more that just json when you run the CLI see [comment here](https://github.com/google/fhir-data-pipes/pull/1433#pullrequestreview-3189992314) . Where it has null would be the actual JSON we are interested in.